src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -243,8 +243,9 @@
                                           (<= input-num-value 0)
                                           (> input-num-value (current-limit)))
             amount-text               (str @input-value " " token-symbol)
+            first-route               (first route)
```

Comment:
We use the first route to grab the native token symbol, should be the same token for all the routes as long as we only support L2s and mainnet (and not sidechains or alternative chains), so using the first one is fine.